DigitalLearningSolutions.Web/ViewModels/LearningMenu/InitialMenuViewModel.cs

```diff
@@ -39,5 +41,21 @@ public InitialMenuViewModel(CourseContent courseContent)
                 courseContent.TutorialsCompletionThreshold
             );
         }
+
+        private static string FormatDuration(int duration)
+        {
+            if (duration < 60)
+            {
+                return FormatTimeUnits(duration, "minute");
+            }
+
+            var formattedHours = FormatTimeUnits(duration / 60, "hour");
+            var formattedMinutes = FormatTimeUnits(duration % 60, "minute");
+            return duration % 60 == 0 ? formattedHours : formattedHours + formattedMinutes;

```

Comment:
Can extract `duration % 60` to a variable? 🙂

Reply:
Yep! Done in 1341b0f 🙂